ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActvitiy.java

```diff
@@ -71,26 +74,33 @@ public void onClick(View v) {
     }
 
     private void save(int tempzero, int tempfive, int tempten) {
-        settingUp(tempzero, tempfive, tempten, new AsyncProcessesCallback() {
-            @Override
-            public void onAsyncSuccess() {
-                addEntry(entryZero);
-                addEntry(entryFive);
-                addEntry(entryTen);
-            }
-        });
-
-
 
-    }
+        entryZero = new ComfortLevelEntry(user, tempzero, 0);
```

Comment:
Can you initialize 0, 5, 10 as private static constants in this class? You can name it as START_COMFOR_LEVEL, MID_COMFORT_LEVEL, END_COMFORT_LEVEL